mmdet/models/registry.py

```diff
@@ -22,9 +22,7 @@ def _register_module(self, module_class):
             module (:obj:`nn.Module`): Module to be registered.
         """
         if not issubclass(module_class, nn.Module):
-            raise TypeError(
-                'module must be a child of nn.Module, but got {}'.format(
-                    type(module_class)))
+            raise TypeError(f'module must be a child of nn.Module, but got {module_class}')
```

Comment:
To be compatible with python 3.4/3.5, we do not use f-strings in this project. When we discard the support for old python versions some day, we will consider introducing some new features.